src/cf-forms/src/atoms/select.less

```diff
@@ -0,0 +1,68 @@
+.a-select {
+    position: relative;
+    border: 1px solid @select-border;
+
+    select {
+        height: unit( 30px / @base-font-size-px, em );
+        width: 100%;
+        padding: unit( 4px / @base-font-size-px, em )
+                 unit( 6px / @base-font-size-px, em );
+        border: 0;
+        outline: 2px solid transparent;
```

Comment:
Is this needed? Since it doesn't affect flow I think you can limit it to hover and focus.